schema/workflow.yaml

```diff
@@ -325,357 +394,435 @@ $defs:
           with:
             type: object
             additionalProperties: true
+            title: CallFunctionWith
```

Comment:
To be consistent, this should follow previous naming convention. If we apply my previous comments, this should then be called FunctionArguments or something like that